bentoml/_internal/configuration/default_configuration.yaml

```diff
@@ -41,6 +41,7 @@ runners:
 tracing:
   type: zipkin
   sample_rate: 0.0
+  excluded_urls: Null
```

Comment:
I wonder whether there are others field we also want to exclude?

But I think it is good for 1st pass.

Reply:
Sorry I'm not quite sure what you mean others field we want to exclude - can you explain more?

Reply:
So we can make `excludes` as a dictionary. 
```
exclude:
  - urls: ...
  - .... <???>
```

I'm not to familiar with tracing, hence I'm wondering if there are any other options people also want to exclude in addition to urls.

Reply:
I am also not too familiar with tracing myself, but it looks like at least for now, the only thing you can exclude in opentelemetry instrumentation is urls...

Reply:
sg